services.yaml

```diff
@@ -665,21 +660,21 @@ paths:
       tags:
         - Events
       summary: "Fetch event actions"
-      description: "Fetch event actions"
+      description: "Get the actions behind the event"
       parameters:
         - in: path
           name: env_id
-          description: Environment ID to use (e.g. S1234)
+          description: Specify the Environment ID (e.g., S1234)
           type: string
           required: true
         - in: path
           name: event_id
-          description: Event to use
+          description: Specify the Environment ID (e.g., S1234)
```

Comment:
event ID

Reply:
@jenjenp doesn't seem like all of this is resolved

Reply:
resolved now